src/BlackSlope.Api.Common/Swagger/DocumentFilterAddHealth.cs

```diff
@@ -7,7 +7,7 @@ namespace BlackSlope.Api.Common.Swagger
     public class DocumentFilterAddHealth : IDocumentFilter
     {
         public void Apply(SwaggerDocument swaggerDoc, DocumentFilterContext context) =>
-            swaggerDoc.Paths.Add("/health", HealthPathItem());
+            swaggerDoc?.Paths.Add("/health", HealthPathItem());
```

Comment:
Did you notice an exception thrown with this code at some point?

Consider the original code as we probably want this to throw or this might become a silent error.

Reply:
No, was just getting rid of Analyzer warning from CA1062 but yes silent error in this case is even worst, so I can just add a Contract.Requires call and leave this without '?.'